dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java

```diff
@@ -331,6 +331,8 @@ public Integer export() throws Exception {
         var jkubeVersion = jkubeMavenPluginVersion(settings, mapBuildProperties());
         buildProperties.add("jkube.version=%s".formatted(jkubeVersion));
 
+        setContainerHealthPaths();
```

Comment:
Why do you need to do this? It defeats the purpose of the component. Ideally you need to include those probes on the Deployment without worrying which is the runtime used.

Reply:
jkube relies on quarkus and sb properties to generate the various probes, look for jkube-healthcheck-spring-boot and jkube-healthcheck-quarkus in jkube maven plugin doc. That's why I used it so jkube generates the container probes manifests accordingly.